src/libraries/System.Data.Common/src/System/Data/DataSet.cs

```diff
@@ -380,8 +383,10 @@ private void DeserializeDataSetSchema(SerializationInfo info, StreamingContext c
                         byte[] buffer = (byte[])info.GetValue(string.Format(CultureInfo.InvariantCulture, "DataSet.Tables_{0}", i), typeof(byte[]));
                         MemoryStream memStream = new MemoryStream(buffer);
                         memStream.Position = 0;
+#pragma warning disable MSLIB0003 // Issue https://github.com/dotnet/runtime/issues/39289 tracks finding an alternative to BinaryFormatter
```

Comment:
Nit: above you only surround the Serialize; here you surround the construction and the deserialize... it'd be nice to be consistent in the tightness wherever possible

Reply:
Tightened it to surround only the obsolete API, not the ctor. Thanks for the tip!